lib/util/util.js

```diff
@@ -193,32 +186,32 @@ const Util = {
                                     solutionSet.add('Apply manual fix in your config.');
                                 }
                                 if (
-                                    !properties.credentials[cred].clientId ||
-                                    properties.credentials[cred].clientId === '--- update me ---'
+                                    !properties.credentials[cred].client_id ||
+                                    properties.credentials[cred].client_id === '--- update me ---'
                                 ) {
-                                    errorMsgs.push(`invalid ClientId on ${cred}`);
+                                    errorMsgs.push(`invalid Client Id on ${cred}`);
                                     solutionSet.add(`Run 'mcdev init ${cred}'`);
                                 }
                                 if (
-                                    !properties.credentials[cred].clientSecret ||
-                                    properties.credentials[cred].clientSecret ===
+                                    !properties.credentials[cred].client_secret ||
+                                    properties.credentials[cred].client_secret ===
                                         '--- update me ---'
                                 ) {
-                                    errorMsgs.push(`invalid ClientSecret on ${cred}`);
+                                    errorMsgs.push(`invalid Client Secret on ${cred}`);
                                     solutionSet.add(`Run 'mcdev init ${cred}'`);
                                 }
                                 if (
-                                    !properties.credentials[cred].tenant ||
-                                    properties.credentials[cred].tenant === '--- update me ---'
+                                    !properties.credentials[cred].auth_url ||
+                                    properties.credentials[cred].auth_url === '--- update me ---'
```

Comment:
why the full auth url instead of tenant subdomain? are you planning to save that also going forward?

Reply:
auth url is easier to maintain because its copy paste. less parsing :) we dont have a need to have the tenant apart